app/assets/stylesheets/gds_frontend_overrule.scss

```diff
@@ -0,0 +1,27 @@
+$light-blue-50: #929191;
+
+.form-control {
+    width: 100%
+}
+
+.button-return {
```

Comment:
When I suggested putting this in this gds_frontend_overrule - it was because you were modifying gds stuff in a way - but that was when you were using .button-secondary - .button-return seems like an application class rather than any form of gds stuff ?